release.sh

```diff
@@ -90,7 +90,7 @@ update_requirements() {
 update_version () {
 	# update and commit local version file used by tracking telemetry
 	echo -e "\\nWriting version file..."
-	sed -i "s/^VERSION *=.*/VERSION = '${new_version}'/" ${VERSION_FILE}
+	sed -i "s/^__version__ *=.*/__version__ = '${new_version}'/" ${VERSION_FILE}
```

Comment:
Ah, we also need to fix the `Makefile` now, to check the `_version.py` file, rather than the `setup.py`

https://github.com/Learnosity/learnosity-sdk-python/blob/master/Makefile#L50

Reply:
Oh yeah, missed that one, thanks! Will update this line -
https://github.com/Learnosity/learnosity-sdk-python/blob/b53f29676067c3dee569be33fab80ec54f6d7b42/Makefile#L50

Reply:
Yup, and this one so as to not confuse people https://github.com/Learnosity/learnosity-sdk-python/blob/b53f29676067c3dee569be33fab80ec54f6d7b42/Makefile#L54